katdal/test/test_dataset.py

```diff
@@ -38,3 +94,48 @@ def test_selection_to_list():
     assert_equal(_selection_to_list(1), [1])
     # Groups
     assert_equal(_selection_to_list('all', all=['a', 'b']), ['a', 'b'])
+
+
+class TestVirtualSensors(object):
+    def setup(self):
+        self.target = Target('PKS1934-638, radec, 19:39, -63:42')
+        self.antennas = [Antenna('m000, -30:42:39.8, 21:26:38.0, 1086.6, 13.5, '
+                                 '-8.264 -207.29 8.5965'),
+                         Antenna('m063, -30:42:39.8, 21:26:38.0, 1086.6, 13.5, '
+                                 '-3419.5845 -1840.48 16.3825')]
+        corrprods = [('m000h', 'm000h'), ('m000v', 'm000v'),
+                     ('m063h', 'm063h'), ('m063v', 'm063v'),
+                     ('m000h', 'm063h'), ('m000v', 'm063v')]
+        subarray = Subarray(self.antennas, corrprods)
+        spw = SpectralWindow(centre_freq=1284e6, channel_width=0, num_chans=16,
+                             sideband=1, bandwidth=856e6)
+        self.timestamps = 1234667890.0 + 1.0 * np.arange(10)
```

Comment:
Did you meant to use 1234567890.0? Or do you just have something against 5's? :-)

Reply:
I had the 5 there but then the source was below the horizon and I fiddled with the timestamp until it rose 😄 I figured that would be a more representative data set.